src/components/Workspace/Workspace.tsx

```diff
@@ -63,10 +62,17 @@ export interface StatusData {
 export async function fetchProblemData(
   problemID: string
```

Comment:
Oops this isn't your fault, but this function should really be in `pages/api/createUSACOFile.ts`, not `Workspace.tsx`. my bad...